CODEOWNERS

```
@@ -141,6 +141,9 @@ packages/app-bridge/src/window.d.ts @Frontify/content-and-blocks @Frontify/theme
 packages/cli/src/commands/ @Frontify/content-and-blocks @Frontify/themes-and-navigation @Frontify/apps-and-integrations
 packages/cli/src/errors/ @Frontify/content-and-blocks @Frontify/themes-and-navigation @Frontify/apps-and-integrations
 packages/cli/src/utils/ @Frontify/content-and-blocks @Frontify/themes-and-navigation @Frontify/apps-and-integrations
+packages/cli/src/utils/platformAppCompiler.ts @Frontify/apps-and-integrations
```

```suggestion
packages/cli/src/utils/compilePlatformApp.ts @Frontify/apps-and-integrations
```

So its the same the others?